languages/rust/exercises/concept/numbers/.meta/design.md

```diff
@@ -0,0 +1,49 @@
+# Design
+
+## Goal
+
+The goal of this exercise is to teach how numbers are implemented in Rust. Because Rust doesn't have implicit casting, this automatically includes teaching about primitive bit sizes. 
```

Comment:
"primitive bit sizes" seems like an accurate description but perhaps we add first that we are introducing students to type casting?